app/controllers/lifehacks_controller.rb

```diff
@@ -7,4 +7,32 @@ def show
     @lifehack = Lifehack.find(params[:id])
     @reviews = @lifehack.reviews
   end
+
+  def new
+    if user_signed_in?
+      @lifehack = Lifehack.new
+    else
+      flash[:error] = "You must be signed in to add a lifehack!"
+      redirect_to lifehacks_path
+    end
+  end
+
+  def create
+    @lifehack = Lifehack.new(lifehack_params)
+    @user = current_user
+    @lifehack.creator = @user
+    if @lifehack.save
+      redirect_to lifehack_path(@lifehack),
+                  notice: 'Lifehack Created Successfully!'
+    else
+      flash.now[:error] = "error in submission"
+      render :new
+    end
+  end
+
+  private
+
+  def lifehack_params
+    params.require(:lifehack).permit(:title, :description, :creator_id)
```

Comment:
It looks like the creator id is being assigned on like 23 so you actually are not permitting that id from the params so you should be able to remove `:creator_id` from this list.